src/app/tasks/tasks.service.ts

```diff
@@ -85,4 +89,15 @@ export class TasksService {
     );
   }
 
+  removeAssigneeFromTask(assignee: any = '') {
+    const removeAssignee = assignee._id === assignee.userId;
+    const link = `/teams/view/${this.link.teams}`;
+
+    if (removeAssignee) {
+      return this.couchService.findAll(this.dbName, findDocuments(this.getTasks())).pipe(
```

Comment:
`findDocuments` is a function for creating a query in the CouchDB [Mango language](https://docs.couchdb.org/en/stable/api/database/find.html?highlight=find#selector-syntax).  You only need the first parameter, the selector, and it should be:
```
{
  assignee,
  link: { team: teamId }
}
```